_search-plugins/search-pipelines/filter-query-processor.md

```diff
@@ -23,6 +20,7 @@ Field | Data type | Description
 `query` | Object | A query in query domain-specific language (DSL). For a list of OpenSearch query types, see [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/). Required. 
 `tag` | String | The processor's identifier. Optional.
 `description` | String | A description of the processor. Optional.
+`ignore_failure` | Boolean | If `true`, OpenSearch [ignores a failure]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/#ignoring-processor-failures) of this processor and continues to run the remaining processors in the search pipeline. Optional. Default is `false`.
```

Comment:
@mingshl Boolean here does not refer to the Java class vs. the primitive type. It's the general concept of a boolean, and it's capitalized because it's capitalized as a word.

Reply:
make sense! Thanks